CMR/python/cmr/search/granule.py

```diff
@@ -72,6 +73,57 @@ def granule_core_fields(item):
     record['native-id'] = meta.get('native-id')
     return {key: value for key, value in record.items() if value}
 
+def _collection_sample_limits(limit:int, climit:int, glimit:int):
+    """
+    Calculate out how many collections and granules should be used to satisfy
+    a requested sample. Allow for a simple case and a more defined case as such:
+
+    1. If a limit is provided, grab 10 granules from a set of collections
+
+    2. If no limit is given, then allow for a client and granule limit to be
+        specified and make assumptions if they are not.
+
+    """
+    if limit is None:
+        c_limit = 10 if climit is None else min(max(1, climit),100)
```

Comment:
Don't understand this min(max(1, climit),100) logic.  max(1,climit) will not be more than 1, swhy do we have to do min(1, 100)?  why don't we just use max(1, climit)?